clap_blocks/src/compactor2.rs

```diff
@@ -178,8 +195,13 @@ pub struct Compactor2Config {
     )]
     pub max_num_files_per_plan: usize,
 
-    /// Maximum input bytes (in parquet) per partition. If there is more data, we ignore the partition (for now) as a
-    /// self-protection mechanism.
+    /// Maximum input bytes (in parquet) per partition.
+    ///
+    /// If the sum of the sizes of all parquet files in a partition is greater than
+    /// this value, the compactor will not try to compact this partition.
```

Comment:
```suggestion
    /// If the sum of the sizes of `files_to_compact` is greater than
    /// this value, the compactor will not try to compact this partition.
```

and I think you want to define `files_to_compact` but I know you know what it is. Number of `files_to_compact` is usually a lot smaller than number of files in the partition. That is why we do not hit this case often after we turn `TargetLevel` on

Reply:
Since `files_to_compact` is some part of the internal calculation of the compactor and will not be easy for users to compute. 

I will try and clarify this wording to make it clear it relates to the worst case (when all files are being compacted together)

Reply:
Clarified in f1820732e